.github/workflows/build-images-pr.yaml

```
@@ -0,0 +1,15 @@
+name: PR Conditional Build and Push Image to Quay.io Repo
+
+on:
+  pull_request_target:
+    types: [labeled, opened, synchronize, reopened]
+
+jobs:
+  workflow-build:
+    if: ${{ github.event_name == 'pull_request_target' && contains(github.event.pull_request.labels.*.name, 'forked_image_approved') }}
```

```suggestion
    if: contains(github.event.pull_request.labels.*.name, 'forked_image_approved')
```

I think we can simplify to this instead since we should only get events from `pull_request_target` defined in the `on` previously already